src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/filingpackage/service/FilingPackageServiceImpl.java

```diff
@@ -39,6 +39,8 @@ public FilingPackageServiceImpl(EfilingReviewService efilingReviewService, Accou
 
         if (!filingPackage.isPresent()) return Optional.empty();
 
+        filingPackage.get().getDocuments().removeIf(document -> document.getStatusCode().equals(ca.bc.gov.open.jag.efilingcommons.Keys.WITHDRAWN_STATUS_CD));
```

Comment:
Can we use `StringUtils.equals()` string.equals is not null safe

Reply:
I like the removeif though

Reply:
Yeah that is a nice function